dashboard_docs.py

```
@@ -0,0 +1,389 @@
+    }
+    requests.append(bullet_req)
+    return requests, current_index + lines_length
+
+
+def generate_detailed_pr_summaries_section(prs, start_index):
+    """
+    Create requests to insert "Detailed PR Summaries" heading (as a heading style),
+    then each PR summary as a sub-heading (HEADING_3), plus normal text.
+    Returns (list_of_requests, new_index).
+    """
+    requests = []
```

Ruff (0.8.2)

Redefinition of unused `requests` from line 3

(F811)